docs/Migrating.md

```diff
@@ -105,6 +105,34 @@ you wish to use a single behavior to work with multiple board sizes, override `G
 current `BoardSize`. The values returned by `GetCurrentBoardSize()` must be less than or equal to the corresponding
 values from `GetMaxBoardSize()`.
 
+### GridSensor changes
+The sensor configuration has changed:
+* The sensor implementation has been refactored and exsisting GridSensor created from extension package
+will not work in newer version. Some errors might show up when loading the old sensor in the scene.
+You'll need to remove the old sensor and create a new GridSensor.
+* These parameters names have changed but still refer to the same concept in the sensor: `GridNumSide` -> `GridSize`,
+`RotateToAgent` -> `RotateWithAgent`, `ObserveMask` -> `ColliderMask`, `DetectableObjects` -> `DetectableTags`
+* `RootReference` is removed and default to the sensor component's GameObject.
+* `DepthType` (`ChanelBase`/`ChannelHot`) option and `ChannelDepth` are removed. Now the default is
+one-hot encoding for detected tag. If you were using original GridSensor without overriding any method,
+switching to new GridSensor will produce similar effect for training although the actual observations
+will be slightly different.
+
+For creating your GridSensor implementation with custom data:
+* To create custom GridSensor, derive from `GridSensorBase` instead of `GridSensor`. Besides overriding
```

Comment:
This should go in the documentation somewhere (and refer to it from the migration guide)